packages/runner/injection/patches/cookies.ts

```
@@ -63,6 +63,17 @@ export const patchDocumentCookie = (requestCookies: AutomationCookie[]) => {
     })
   }
 
+  const setCookie = (cookie: ToughCookie | string) => {
+    try {
+      return cookieJar.setCookie(cookie, url, undefined)
+    } catch (err) {
```

Should we propagate this error so users are aware it wasn't properly set?

---

No, it should be ignored. That's what the real `document.cookie` would do.

---

Sure, I just seem like if that' be obvious from the commands that tried to set this